src/chromo/util.py

```diff
@@ -396,6 +398,13 @@ def _download_file(outfile, url):
         raise ConnectionError(f"{fname} has not been downloaded")
 
 
+# Path to directory to store precomputed data tables
+def _cache_base_dir():
```

Comment:
No docstring :)

Reply:
If you insist on a docstring, we can of course add one, but I usually don't add docstrings for private functions (starting with underscore), because these are not to be called by users and therefore don't need user-friendly documentation. Linters should not complain about those.

Reply:
It's ok :)